src/util/output/link.ts

```diff
@@ -0,0 +1,5 @@
+import chalk from 'chalk';
+
+const link = (text: string) => chalk.cyan.underline(text);
```

Comment:
I think this can be simplified:

```suggestion
const link = chalk.cyan.underline;
```